cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/WorkflowDataset.java

```diff
@@ -238,7 +313,7 @@ public ProgramType getProgramType() {
   /**
    * Internal class to keep track of Workflow Run Records
    */
-  static class WorkflowRunRecord {
+  public static class WorkflowRunRecord {
```

Comment:
then remove the comment about it being an internal class, otherwise move it out from this class and make it non-static.